xwiki-platform-core/xwiki-platform-livetable/xwiki-platform-livetable-ui/src/main/resources/XWiki/LiveTableResultsMacros.xml

```diff
@@ -881,33 +881,6 @@
 #end
 
 
-#macro (parseDateRange $matchType $filterValue $dateRange)
```

Comment:
I hope nobody reuse this macro in custom livetable sources (it's the case of many of the macros in that page).